client/src/components/signing-form/signing-form.html

```diff
@@ -0,0 +1,37 @@
+<div class="signing-form">
+    <div data-bind="if: !isSignUp()">
+        <h1 class="signing-form__title">Sign in</h1>
+    </div>
+    <div data-bind="if: isSignUp()">
+        <h1 class="signing-form__title">Sign up</h1>
+    </div>
+
+    <div class="signing-form__group">
+        <input placeholder="email" data-bind="textInput: email, css: {'signing-form__submit--invalid': email.isValid()}"
```

Comment:
`textInput` binding seems to be validatable by default, so all you had to do was specify validation class in ko validation settings.